src/DBus/DBusBridge.cpp

```diff
@@ -93,7 +97,11 @@ namespace usbguard
     }
 
     if (method_name == "setParameter") {
-      if (! isAuthorizedByPolkit(invocation)) {
+      GDBusError authErrorCode = G_DBUS_ERROR_FAILED;
```

Comment:
Would it be possible to bundle the following lines? Doing so we could remove duplicate code as these occur many times.

Reply:
I don't really see how, at least not in C++. Any concrete ideas how to?

Reply:
I can see two possibilities here:

- C++ solution what @benzea already suggested (https://github.com/USBGuard/usbguard/pull/548#issuecomment-1148431719) or 
- a C-like solution using macros
Anyway, this is not a blocker :) I think it can be done in a later PR